Jasmine/Common/Helpers/Random.swift

```diff
@@ -14,6 +14,15 @@ enum Random {
         return from + Int(arc4random_uniform(UInt32(toInclusive - from + 1)))
     }
 
+    /// Generates a random integer
+    ///
+    /// - Parameters:
+    ///   - toInclusive: end of range, inclusive
+    /// - Returns: a random number from 0 to toInclusive
+    static func integer(toInclusive: UInt) -> Int {
```

Comment:
if you don't include the `inclusive` in the argument label, you won't see that it's inclusive in the code hint pop up though.